### PR TITLE
BCD & DI AVT - Need grouping roles or fieldset element to identify related radios org-ids/web-ide-issues#790

### PR DESCRIPTION
**Pull request by squarti**

### CODE REVIEW

File: bundles/org.eclipse.orion.client.git/web/orion/git/widgets/ApplyPatchDialog.js [hunk 1]
```diff
@@ -21,11 +21,13 @@ define([ 'i18n!git/nls/gitmessages', 'orion/webui/dialog', 'orion/xsrfUtils', 'o
 
 	ApplyPatchDialog.prototype.TEMPLATE =
 
+	'<span role="radiogroup" aria-label="${Patch Location}>"' +
```
Comment (carmacleod) [inline bundles/org.eclipse.orion.client.git/web/orion/git/widgets/ApplyPatchDialog.js]: If possible, I would use a div here and not a span. Technically, the [content model for span](http://w3c.github.io/html/textlevel-semantics.html#the-span-element) is to only have [phrasing content](http://w3c.github.io/html/dom.html#phrasing-content-2), which does not include div.

See the Note in the definition of "phrasing content" that says:
> Most elements that are categorized as phrasing content can only contain elements that are themselves categorized as phrasing content, not any flow content.

Comment (squarti) [reply bundles/org.eclipse.orion.client.git/web/orion/git/widgets/ApplyPatchDialog.js]: done

**Review by carmacleod — CHANGES_REQUESTED** [review]
In the Apply Patch dialog, the URL and File labels are not connected to their radio or their text/file input.
Please use aria-labelledby to give each radio a label.
Also use aria-labelledby on the text/file input to give them the same label.
(So you would have URL radio, then type tab and get URL text field).

Or (better way), put the text content in a `<label>` element and hook up the `for` to the id of the radio.
Give the label element an id, and hook up the aria-labelledby of the text/file input to point to the label id.
See the way it was done in the Init Git Repository dialog.

**Review by carmacleod — APPROVED** [review]
+1

**Pull request merged**